js/core/templates/webidl-contiguous/attribute.html

```diff
@@ -1,3 +1,3 @@
-<span class='idlAttribute' id="{{obj.idlId}}" data-idl data-title='{{obj.name}}'>{{extAttr obj indent
-}}{{idn indent}}{{qualifiers}}attribute <span class='idlAttrType'>{{idlType obj}}</span> {{pads
-pad}}<span class='idlAttrName'>{{#tryLink obj}}{{escapeAttributeName obj.name}}{{/tryLink}}</span>;</span>
+<span class='idlAttribute' id="{{obj.idlId}}" data-idl data-title='{{obj.name}}'>{{extAttr obj
+}}{{{qualifiers}}}{{{trivia obj.trivia.base}}}attribute<span class='idlAttrType'>{{idlType
+obj}}</span> <span class='idlAttrName'>{{#tryLink obj}}{{obj.escapedName}}{{/tryLink}}</span>;</span>
```

Comment:
double checked that the new line at EOF is not needed, right? These can be super annoying.

Reply:
Same with the ones below... they look ok, given that they are spans... but just triple check. It's gotten me before /shakes fist at handlebars.